src/core_plugins/kibana/server/tutorials/apm/apm_server_instructions.js

```diff
@@ -0,0 +1,63 @@
+const EDIT_CONFIG = {
+  title: 'Edit the configuration',
```

Comment:
@nreese Does it makes sense to keep these X-Pack instructions in the OSS version?

Reply:
I think so. All of the beats instructions include directions for security

Reply:
++